reference-architecture/vmware-ansible/README.md

```diff
@@ -19,18 +19,19 @@ rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm*
 yum -y install atomic-openshift-utils \
                   git \
                   pyOpenSSL \
-                  python-simplejson \
                   PyYAML \
-                  python-ldap \
                   python-click \
+                  python-httplib2
                   python-iptools \
+                  python-ldap \
                   python-netaddr \
-                  python-httplib2
-git clone https://github.com/dannvix/pySphere && cd pySphere/ && python setup.py install
+                  python-simplejson \
+                  python-six \
+
 git clone https://github.com/vmware/pyvmomi && cd pyvmomi/ && python setup.py install
 
 # Grabbed the patched vsphere_guest until the PR is merged
-cp vsphere_guest.py /usr/lib/python2.7/site-packages/ansible/modules/core/cloud/vmware/
+cp vmware_guest.py /usr/lib/python2.7/site-packages/ansible/modules/core/cloud/vmware/
```

Comment:
From the comment below.

Yea just store it within the repo and don't copy it to the ansible directory. One concern I would have would be overwriting something in the current Ansible directory or if an Ansible patch comes out the module may be overwrote or deleted and that could break us. If we ship it within our repo then we know we have a known good module until upstream makes it to fully supported.